chapter_philosophy_and_refactoring.asciidoc

```diff
@@ -214,6 +218,8 @@ class NewVisitorTest(unittest.TestCase):
 ----
 ====
 
+// DAVID: I'd use ... over [...] in code samples as it's simpler (and still valid Python).
```

Comment:
```python
[...]
```

is valid python too!

not sure if i have the energy to go change these and *all* their test. but otoh, i've got so used to the existing ones that i can't "see" them impartially any more.  maybe this is exactly the reason to have fresh eyes.  arg but so much effort.